r/configure

```diff
@@ -121,16 +121,25 @@ else
   ARROW_USE_PKG_CONFIG="false"
 fi
 
-# find openssl on macos. macOS ships with libressl. openssl is installable
-# with brew, but it is generally not linked. We can over-ride this and find
-# openssl but setting OPENSSL_ROOT_DIR (which cmake will pick up later in
-# the installation process). FWIW, arrow's cmake process uses this
-# same process to find openssl, but doing it now allows us to catch it in
-# nixlibs.R and throw a nicer error.
+## Find openssl
+# Arrow's cmake process uses this same process to find openssl,
+# but doing it now allows us to catch it in
+# nixlibs.R and activate S3 and GCS support for the source build.
+
+# macOS ships with libressl. openssl is installable with brew, but it is 
+# generally not linked. We can over-ride this and find
+# openssl by setting OPENSSL_ROOT_DIR (which cmake will pick up later in
+# the installation process).
 if [ "${OPENSSL_ROOT_DIR}" = "" ] && brew --prefix openssl >/dev/null 2>&1; then
   export OPENSSL_ROOT_DIR="`brew --prefix openssl`"
   export PKG_CONFIG_PATH="${OPENSSL_ROOT_DIR}/lib/pkgconfig${PKG_CONFIG_PATH:+:${PKG_CONFIG_PATH}}"
 fi
+# Look for openssl with pkg-config for non-brew sources(e.g. CRAN) and Linux
```

Comment:
This is very minor

> for non-brew sources(e.g. CRAN)

I'm not sure I follow the connection between non-brew and CRAN here. This would work for anyone who has openssl installed in a way that pkg-config can find it, yeah? It might be clearer to just leave it at that?

Reply:
Yes this would of course work for any setup that has pkg-config available but we know that CRAN does not have brew but does have pkg-config so that's why I list it as an example :) (contrasting to the brew finding logic that's directly above which I would say is the default for the 'normal' mac dev)